charts/openfga/templates/deployment.yaml

```diff
@@ -272,7 +272,7 @@ spec:
 
           readinessProbe:
             exec:
-              command: ["/bin/grpc_health_probe", "-addr={{ .Values.grpc.addr }}"]
+              command: ["grpc_health_probe", "-addr={{ .Values.grpc.addr }}"]
```

Comment:
I think the Chart shouldn't care about the location of the probe, only that it is in PATH. If it was already this way, we wouldn't have had to make this change here.

In case we update the location in the future, the chart shouldn't be concerned with that.